hail/python/hail/experimental/time.py

```diff
@@ -0,0 +1,60 @@
+import hail as hl
+from hail.expr.expressions.expression_typecheck import *
+from hail.expr.functions import _func
+from hail.typecheck import *
+
+@typecheck(format=expr_str,
+           time=expr_int64,
+           zone_id=expr_str)
+def strftime(format, time, zone_id):
```

Comment:
I find it a little confusing that `strftime` and `strptime` take their arguments in different orders. Was that copying from somewhere else? The python `strftime` is a method on a time object, and `strptime` takes the date string first, so the most consistent might be to take the time argument first in both cases.

Reply:
I also think it's confusing. I was trying to copy python:

https://docs.python.org/3.7/library/time.html#time.strptime
https://docs.python.org/3.7/library/time.html#time.strftime

strftime has the format string first, strptime has it second.

Reply:
Oh I missed the static functions. I guess that's so `strptime` can have a default format, and `strftime` can default to the current time.

Disregard, matching python is good.